js/NewsController.js

```diff
@@ -0,0 +1,130 @@
+import {
+    constants
+} from "./data.js";
+
+import {
+    dataFetch
+} from "./NewsModel.js";
+
+import {
+    viewPage
+} from "./NewsView.js";
+
+
+export class postControl {
+    category = "cnn";
+
+
+    init = () => {
+        let obj = new viewPage();
+        obj.renderPage();
+
+    }
+
+    onLoadPage = () => {
+        let obj = new dataFetch(this.category);
+        obj.fetchData();
+
+    }
+
+    setPostData = (sourceOfPost, data) => {
+        let postData = [];
+        //urlToImage,title,publishedAt,description,{postCategory:this.source}
+        data.forEach(function (eachPost) {
+            postData.push({
+                imageSrc: eachPost.urlToImage,
+                postTitle: eachPost.title,
+                postDate: eachPost.publishedAt,
+                postCategory: sourceOfPost,
+                postDesc: eachPost.description
+            })
+        });
+        let obj = new viewPage();
+        obj.setData(postData);
+
+    }
+
+    sourceSelectionEvent = (sourceOfPost, data) => {
+    //let objConst = new constants();
```

Comment:
remove unused code instead of putting them in comments

Reply:
removed unused code